superset/sql_lab.py

```diff
@@ -264,11 +263,7 @@ def execute_sql_statement(  # pylint: disable=too-many-statements
         sql = apply_limit_if_exists(database, increased_limit, query, sql)
 
     # Hook to allow environment-specific mutation (usually comments) to the SQL
-    sql = SQL_QUERY_MUTATOR(
-        sql,
-        security_manager=security_manager,
-        database=database,
-    )
+    sql = database.mutate_sql_based_on_config(sql)
```

Comment:
@john-bodley here's the call to the centralized method that applies the same logic consistently across `SQL_QUERY_MUTATOR` and `MUTATE_AFTER_SPLIT` in one place